### PR TITLE
Blog post for 0.21.1 hotfix: EU consent cookie

### PR DESCRIPTION
**Pull request by opusforlife2**

### CODE REVIEW

File: _posts/2021-04-10-newpipe-0.21.1-released.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+date: 2021-04-10 XX:XX:XX + 1:00
+categories: [pinned, release]
+author: Opusforlife & Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+This is a hotfix release to bypass the new YouTube consent page shown to users in the European Union.
+
+The current workaround is not guaranteed to work forever. It may be deprecated soon. You could say this is just a "warm fix".
```
Comment (TheAssassin) [inline _posts/2021-04-10-newpipe-0.21.1-released.md:14]: > "warm fix"

Pretty confusing neologism. I'd remove the last sentence and create just one paragraph as intro to improve the reading flow.

File: _posts/2021-04-10-newpipe-0.21.1-released.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+No worries, though. The team is working to find a proper long-term solution to the problem.
+
+For the non-EU users: Carry on! Nothing to see here.
+
```
Comment (TheAssassin) [inline _posts/2021-04-10-newpipe-0.21.1-released.md:19]: Please insert a "more" separator.

**Review by TheAssassin — CHANGES_REQUESTED** [review]

File: _posts/2021-04-10-newpipe-0.21.1-released.md (hunk 2 above)
Comment (opusforlife2) [reply _posts/2021-04-10-newpipe-0.21.1-released.md:17]: Done.

File: _posts/2021-04-10-newpipe-0.21.1-released.md (hunk 1 above)
Comment (opusforlife2) [reply _posts/2021-04-10-newpipe-0.21.1-released.md]: Done

**Comment by TheAssassin** [comment]
Merged and fixed in 729ae2cea423346ea4a60b073096af4902f9dfe1.

**Pull request closed without merging**